src/StatusQ/Components/StatusLetterIdenticon.qml

```diff
@@ -0,0 +1,26 @@
+import QtQuick 2.13
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+Rectangle {
+    id: statusLetterIdenticon
+
+    property string name
+    property int letterSize: 21
+
+    color: Theme.palette.miscColor5
+    width: 40
+    height: 40
+    radius: width / 2
+
+    StatusBaseText {
+        text: (statusLetterIdenticon.name.charAt(0) == "#" ? statusLetterIdenticon.name.charAt(1) : statusLetterIdenticon.name.charAt(0)).toUpperCase()
+        font.weight: Font.Bold
+        font.pixelSize: statusLetterIdenticon.letterSize
+        color: Qt.rgba(255, 255, 255, 0.7)
```

Comment:
This will always be 70% white regardless of the theme, so I've hardcoded it here.